graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/exceptions/DuplicateEntityFetcherException.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2024 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.graphql.dgs.exceptions
+
+import java.lang.reflect.Method
+
+class DuplicateEntityFetcherException(
+    val entityType: String,
+    val firstEntityFetcherClass: Class<out Any>,
```

Comment:
what if there are multiple (>=3) entity fetechers? it would be more efficient to have a set or smth and fail when there are more than one so that the consumer doesn't have to restart the app to find all violations

Reply:
That would require extra temporary storage, because there will be only one value per key in the map. This is possible but adds complexity, which I don't think it's worth it because it's already quite an uncommon case.

Reply:
fair